packages/file-search/src/browser/quick-file-open.ts

```diff
@@ -29,18 +38,49 @@ export class QuickFileOpenService implements QuickOpenModel {
     }
 
     private wsRoot: FileStat | undefined;
+    private showIgnored: boolean = false;
+    private opened: boolean = false;
 
     isEnabled(): boolean {
         return this.wsRoot !== undefined;
     }
+    isOpened(): boolean {
+        return this.opened;
+    }
+    isShowIgnored(): boolean {
+        return this.showIgnored;
+    }
 
     open(): void {
-        this.quickOpenService.open(this, {
-            placeholder: 'file name to search',
-            fuzzyMatchLabel: true,
-            fuzzyMatchDescription: true,
-            fuzzySort: true
-        });
+        let placeholderText = "";
+        placeholderText = 'file name to search Press ' + this.getkeyCommand() + ' to toggle Show / Hide .gitignore files';
```

Comment:
Please start the message with a capital letter and remove the capital P in `Press`.  I think we should aim at keeping these placeholders as short and simple as possible.  What about his?

`File name to search (press ${keybinding} to show/hide ignored files).`

Reply:
Done